src/main/main.c

```diff
@@ -280,9 +280,18 @@ void init(void)
 	pwm_params.useMultiShotPwmRate = feature(FEATURE_MULTISHOT_PWM_RATE);
     pwm_params.useFastPWM = masterConfig.use_fast_pwm ? true : false;
     pwm_params.motorPwmRate = masterConfig.motor_pwm_rate;
-    pwm_params.idlePulse = masterConfig.escAndServoConfig.mincommand;
     if (feature(FEATURE_3D))
+    {
         pwm_params.idlePulse = masterConfig.flight3DConfig.neutral3d;
+    }
+    else 
```

Comment:
Lance has tested last night brushed and brushless ONESHOT_PWM_RATE. Looks to be the following works quite well.

if ((pwm_params.motorPwmRate > 500 && !masterConfig.use_fast_pwm)&& !feature(FEATURE_ONESHOT_PWM_RATE) && !feature(FEATURE_MULTISHOT_PWM_RATE))
    pwm_params.idlePulse = 0; // brushed motors